package.json

```diff
@@ -90,7 +91,7 @@
         "file-loader": "^6.2.0",
         "graphql-depth-limit": "^1.1.0",
         "graphql-tools": "^8.2.9",
-        "html-loader": "^3.1.0",
+        "html-loader": "^3.1.2",
         "html-webpack-plugin": "^5.5.0",
         "husky": "^8.0.1",
         "lint-staged": "^12.4.1",
```

Comment:
Can uninstall `url-loader`. Please review for any other unused modules and let me know if you have any questions.

Reply:
Not sure if I can. I reversed back in order to use .png files I needed and it worked?